avr/cores/megacommand/memory.h

```diff
@@ -47,21 +47,18 @@ class RamBankSelector {
 #endif// __cplusplus
 
 extern inline void memcpy_bank1(volatile void *dst, volatile void *src, uint32_t len) {
-  switch_ram_bank(1);
+  select_bank(1);
```

Comment:
I thought these should be equivalent? 🤔

Reply:
Because the ring buffer code is being called within UART ISR, there may be some circumstances where the original bank, before the ISR routine was already bank1.

So switching back to bank 0 will cause things to blow up when the program exits the ISR.

Reply:
There's still something wrong with this code. I was playing around with it late last night.
I'm not sure if it's incorrect bank switching, or memory leak, or something else.

Reply:
oww, I got it; switch_ram_bank(0) wasn't ideal.

Reply:
going over the code.